src/Components/Counter/Counter.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Counter.css'
+
+function Counter({fruitNameCounter, setFruitNameCounter, fruitName, emoticon}) {
+    return (
+        <article className="card">
+            <div className="emoticon">{emoticon} <h2>{fruitName}</h2></div>
+
+            <div>
+                <button
+                    type="button"
+                    onClick={() => setFruitNameCounter(fruitNameCounter - 1)}
```

Comment:
Zodra `fruitNameCounter` veranderd zorgt dit voor een onnodige rerender door een wijziging in deze functie. Je zou ook de waarde kunnen gebruiken je tot je beschikking hebt bínnen de `setFruitNameCounter`:
```suggestion
                    onClick={() => setFruitNameCounter(currentValue => currentValue - 1)}
```